Form - Vue and React samples (#628)
DevExpress/devextreme-documentation: concepts/05 Widgets/Form/20 Change Options at Runtime/05 Widget Options.md

        });
    });

> With Angular, Vue or React, bind the option to change to a component or element property.

Reviewer: ```suggestion
With Angular, Vue, or React, bind the option that should be changed to a component property.
```

        // ...
    })

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div>
            <DxForm
                :form-data="employee"
                :disabled="isFormDisabled" />
            <DxCheckBox
                text="Disable the Form"
                :value.sync="isFormDisabled" />
        </div>
    </template>
    <script>
    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

>     import { DxForm, DxCheckBox } from 'devextreme-vue';

Reviewer: ```suggestion
    import { DxForm } from 'devextreme-vue/form';
    import { DxCheckBox } from 'devextreme-vue/check-box';
```

    const employee = {
        firstName: 'John',
        lastName: 'Heart',
        phone: '+1(213) 555-9392',
        email: 'jheart@dx-email.com'
    };

    export default {
        components: {
            DxForm, DxCheckBox
        },
        data() {
            return {
                employee,
                isFormDisabled: false
            };
        },
    };
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';

    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

>     import { Form, CheckBox } from 'devextreme-react';

Reviewer: ```suggestion
    import { Form } from 'devextreme-react/form';
    import { CheckBox } from 'devextreme-react/check-box';
```

    const employee = {
        firstName: 'John',
        lastName: 'Heart',
        phone: '+1(213) 555-9392',
        email: 'jheart@dx-email.com'
    };

    class App extends React.Component {
        constructor() {
            super();
            this.state = {
                isFormDisabled: false
            };
            this.onCheckBoxValueChanged = this.onCheckBoxValueChanged.bind(this);
        };

        render() {
            return (
                <div>
                    <Form
                        formData={employee}
                        disabled={this.state.isFormDisabled} />
                    <CheckBox
                        text="Disable the Form"
                        value={this.state.isFormDisabled}

>                         onValueChanged={this.onCheckBoxValueChanged } />

Reviewer: ```suggestion
                        onValueChanged={this.onCheckBoxValueChanged} />
```

                </div>
            );
        }

        onCheckBoxValueChanged(e) {

>             this.setState(() => {
>                 return { isFormDisabled: e.value };
>             });

Reviewer: ```suggestion
            this.setState({
                isFormDisabled: e.value
            });
```
